examples/Model.elm

```diff
@@ -173,57 +249,58 @@ update msg model =
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Time.every second Tick
+    Time.every Time.second Tick
 
 
 
 -- Helpers
 
 
-recordToFormData : Record -> Form.Model
+recordToFormData : Record -> FormData
 recordToFormData { id, title, description } =
-    Form.Model
+    FormData
         (Just id)
         (Maybe.withDefault "" title)
         (Maybe.withDefault "" description)
 
 
-encodeFormData : Form.Model -> Encode.Value
+encodeFormData : FormData -> Encode.Value
 encodeFormData { title, description } =
     Encode.object
         [ ( "title", Encode.string title )
         , ( "description", Encode.string description )
         ]
 
 
-removeRecordFromList : Record -> Records -> Records
+removeRecordFromList : Record -> List Record -> List Record
 removeRecordFromList { id } records =
-    Dict.remove id records
+    List.filter (\record -> record.id /= id) records
```

Comment:
Dicts are ordered by keys (at least when you convert them to a list, they are).
We wanted to get rid of the dict because it involved a lot of hassle (converting to/from lists all the time, ordering them...). They did make the updating/deleting easier, but that was the only advantage.